experimenter/experimenter/experiments/api/v6/serializers.py

```diff
@@ -205,3 +207,8 @@ def get_localizations(self, obj):
         if obj.is_localized:
             with contextlib.suppress(json.JSONDecodeError):
                 return json.loads(obj.localizations)
+
+    def get_locales(self, obj):
+        locale_codes = [locale.code for locale in obj.locales.all()]
+        if len(locale_codes):
+            return locale_codes
```

Comment:
Don't actually need to do len here.  In theory could just write this as

```py
return [....] or None
```

if we care about null over the empty list?  Or just `return [...]`?

Would be nice if Python had a `?` operator